lib/server/db/postgres/postgres.go

```diff
@@ -105,6 +106,18 @@ func (p *Postgres) PingContext(ctx context.Context) error {
 	return nil
 }
 
+// Exists determines if the database has already been created
```

Comment:
Having this up here doesn't quite align with the debug messages issued by `createDatabase` below. Perhaps the call to should be moved into `createDatabase`?

Reply:
Agreed